build/source/engine/updateVars.f90

```diff
@@ -628,9 +628,14 @@ subroutine updateVars(&
   ! only for soil
   if(ixDomainType==iname_soil)then
 
-   ! check liquid water
-   if(mLayerVolFracLiqTrial(iLayer) > theta_sat(ixControlIndex) )then
+   ! check liquid water (include tolerance)
+   if(mLayerVolFracLiqTrial(iLayer) > theta_sat(ixControlIndex)+epsT )then
     message=trim(message)//'liquid water greater than porosity'
+    print*,'---------------'
+    print*,'porosity(theta_sat)=', theta_sat(ixControlIndex)
+    print*,'liq water =',mLayerVolFracLiqTrial(iLayer)
+    print*,'layer =',iLayer
+    print*,'---------------'
     err=20; return
    endif
 
```

Comment:
Does this have anything to do with the runoff calculation?

Reply:
no, this was an earlier change I never added, meant to solve a case where the tolerances were stopping the run, and to provide some info.  If there's too much info (cluttering the code), the added print statements can be removed.  The epsT did help, though.

Reply:
OK.